PWGCF/FemtoWorld/Tasks/femtoWorldPairTaskTrackPhi.cxx

```diff
@@ -128,29 +128,33 @@ struct femtoWorldPairTaskTrackPhi {
                         o2::aod::FemtoWorldParticles& parts)
   {
     const auto& magFieldTesla = col.magField();
-
     auto groupPartsOne = partsOne->sliceByCached(aod::femtoworldparticle::femtoWorldCollisionId, col.globalIndex());
     auto groupPartsTwo = partsTwo->sliceByCached(aod::femtoworldparticle::femtoWorldCollisionId, col.globalIndex());
     const int multCol = col.multV0M();
     eventHisto.fillQA(col);
     /// Histogramming same event
-    for (auto& part : groupPartsOne) {
+    for (auto& part : groupPartsOne) { // jakby tu nie wchodzilo
+      /*
       if (part.p() > cfgCutTable->get("PartOne", "MaxP") || part.pt() > cfgCutTable->get("PartOne", "MaxPt")) {
         continue;
       }
       if (!isFullPIDSelected(part.pidcut(), part.p(), cfgCutTable->get("PartOne", "PIDthr"), vPIDPartOne, cfgNspecies, kNsigma, cfgCutTable->get("PartOne", "nSigmaTPC"), cfgCutTable->get("PartOne", "nSigmaTPCTOF"))) {
         continue;
-      }
+      }*/
+      // LOGF(info, "TEST ==============================TRACKONEFILL=================================== \n");
       trackHistoPartOne.fillQA(part);
     }
     for (auto& part : groupPartsTwo) {
+      // LOGF(info, "TEST ==============================TRACKPHIFILL=================================== \n");
       trackHistoPartTwo.fillQA(part);
     }
     /// Now build the combinations
     for (auto& [p1, p2] : combinations(groupPartsOne, groupPartsTwo)) {
+      /*
       if (p1.p() > cfgCutTable->get("PartOne", "MaxP") || p1.pt() > cfgCutTable->get("PartOne", "MaxPt")) {
         continue;
       }
+
       if (!isFullPIDSelected(p1.pidcut(), p1.p(), cfgCutTable->get("PartOne", "PIDthr"), vPIDPartOne, cfgNspecies, kNsigma, cfgCutTable->get("PartOne", "nSigmaTPC"), cfgCutTable->get("PartOne", "nSigmaTPCTOF"))) {
         continue;
       }
```

Comment:
Same as above

Reply:
This was commented only temporarily. In next PR I will be adding cuts to the histograms